shell/lint.sh

```diff
@@ -1,25 +1,35 @@
 #!/bin/bash
-isort --sl -c --profile=black .
+files="."
```

Comment:
can you explain this change a bit more thoroughly - maybe through comments?  

Add a usage comment perhaps, showing both ways to use the script.

Reply:
Do you mean files variabile handling?

Reply:
Now that there are two ways to execute the script we should have a comment at the top indicating how to use the script.

Reply:
Done